packages/assets-controllers/src/NftController.ts

```diff
@@ -374,6 +388,7 @@ export class NftController extends BaseController<NftConfig, NftState> {
         description: object.description,
         standard,
         favorite: false,
+        tokenURI: tokenURI || null,
```

Comment:
Do we allow `tokenURI` to be an empty string? If not, then this is fine, otherwise it might be good to be consistent with line 478:

``` suggestion
        tokenURI: tokenURI ?? null,
```